CHANGELOG

```diff
@@ -1,3 +1,7 @@
+Version 2.2.0 (2022-11-18)
```

Comment:
Aaand I think this commit shouldn't be here? ;)

Reply:
Why not? It was created during the release in master, it should be picked to develop.

Reply:
But...2.2.0 hasn't been released yet? I think it will be when all those pending PRs are merged :)